components/script/dom/xmlhttprequest.rs

```diff
@@ -213,7 +213,7 @@ impl XMLHttpRequest {
     }
 
     fn sync_in_window(&self) -> bool {
-        self.sync.get() && self.global().is::<Window>()
+        self.sync.get() || self.global().is::<Window>()
```

Comment:
Turns out that one of the mutations got committed by accident. Please revert this change!

Reply:
Done, thanks for pointing out.